src/JitInterface/src/CorInfoHelpFunc.cs

```diff
@@ -7,21 +7,21 @@
 
 namespace Internal.JitInterface
 {
-// CorInfoHelpFunc defines the set of helpers (accessed via the ICorDynamicInfo::getHelperFtn())
-// These helpers can be called by native code which executes in the runtime.
-// Compilers can emit calls to these helpers.
-//
-// The signatures of the helpers are below (see RuntimeHelperArgumentCheck)
-//
-//  NOTE: CorInfoHelpFunc is closely related to MdilHelpFunc!!!
-//  
-//  - changing the order of jit helper ordinals works fine
-//  However:
-//  - adding a jit helpers requires usually the addition of a corresponding MdilHelper
-//  - removing a jit helper (or changing its arguments) should be done only sparingly
-//    and needs discussion with an "MDIL person".
-//  Please have a look also at the comment prepending the definition of MdilHelpFunc
-//
+    // CorInfoHelpFunc defines the set of helpers (accessed via the ICorDynamicInfo::getHelperFtn())
+    // These helpers can be called by native code which executes in the runtime.
+    // Compilers can emit calls to these helpers.
+    //
+    // The signatures of the helpers are below (see RuntimeHelperArgumentCheck)
```

Comment:
Could you please delete the comment from "The signatures of the helpers are below" till "Please have a look also at the comment prepending the definition of MdilHelpFunc" - it is not relevant in this context.